### PR TITLE
Api

### PR DESCRIPTION
**Pull request by erezfree29**
Milestone 2 working with Api

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
     filename: 'bundle.js',
     path: path.resolve(__dirname, 'dist'),
   },
```
Comment (rloterh) [inline webpack.config.js:9]: - [ ] OPTIONAL
Your webpack configuration does not have `mode` option set. As a best practice kindly ensure that the `mode` option is set to either `development` during build phase or `production` during repository host for production code. The mode option should be inserted within the `module.exports` array as can be seen below:

image

File: src/scripts/index.js [hunk 2]
```diff
@@ -1 +1,89 @@
 import '../style.css';
+
+const fetch = require('node-fetch');
+
+function fillTable(data) {
```
Comment (rloterh) [inline src/scripts/index.js]: - [ ] Your function `fillTable` is not defined using the **arrow function** syntax. We actually want **all** functions defined in this project to make use of the arrow function syntax. Kindly ensure to improve upon this aspect.

File: src/scripts/index.js [hunk 3]
```diff
@@ -1 +1,89 @@
...
+  const fhighestScore = document.querySelector('.fhighest_score');
+  fhighestUser.textContent = Object.values(data)[0][4].user;
+  fhighestScore.textContent = Object.values(data)[0][4].score;
+  const thighestUser = document.querySelector('.thighest');
+  const thighestScore = document.querySelector('.thighest_score');
+  thighestUser.textContent = Object.values(data)[0][5].user;
+  thighestScore.textContent = Object.values(data)[0][5].score;
+}
+
+fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
```
Comment (rloterh) [inline src/scripts/index.js:51]: - [ ] Your data/ API's are not being fetched using the `async` and the `await` methods. To ensure a correct implementation of asynchronous functions we want to fetch our API using the `async` and `await` appropriately together handling them with the `try/catch` block. Kindly ensure to improve upon this aspect in our code at **all** the places data is being fetched

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Erez , 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Data is fetching correctly
- No linter errors


## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 _Check the [OPTIONAL] comment(s) under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**